api/envoy/data/accesslog/v2/accesslog.proto

```diff
@@ -171,6 +171,12 @@ message AccessLogCommon {
   // state serialized to any message other than `google.protobuf.Any` it will be packed into
   // `google.protobuf.Any`.
   map<string, google.protobuf.Any> filter_state_objects = 21;
+
+  // If downstream connection in listenner failed due to transport socket (e.g. TLS handshake), provides the
+  // failure reason from the transport socket. The format of this field depends on the configured
+  // upstream transport socket. Common TLS failures are in
```

Comment:
Why does the downstream transport failure depend on upstream socket selection?

Reply:
Ops, thanks for catching this. It depends on listener format setting I think.